cmd/infra/powervs/validate.go

```diff
@@ -132,6 +132,31 @@ func validateVpc(vpcName string, resourceGroupID string, v1 *vpcv1.VpcV1) (vpc *
 	}
 
 	if vpc != nil {
```

Comment:
it will be better if you handle the err early, e.g:

```golang
if vpc == nil {
        err = fmt.Errorf("%s vpc not found", vpcName)
	return
}

Reply:
Handled vpc validation part early.